dependencies.gradle

```diff
@@ -1,20 +1,20 @@
 dependencies {
     compile("com.github.GTNewHorizons:GT5-Unofficial:master-SNAPSHOT:dev")
 
-    compileOnly("com.github.GTNewHorizons:AppleCore:master-SNAPSHOT:dev") {
+    compile("com.github.GTNewHorizons:AppleCore:master-SNAPSHOT:dev") {
         transitive = false
     }
     compileOnly("com.github.GTNewHorizons:Railcraft:master-SNAPSHOT:dev") {
-        transitive = false
+        transitive = true
```

Comment:
Why true? If you want the dependecy for compilation you should state them explicitly. Keep it false

Reply:
The problem is getting it to run - compileOnly builds but then runClient explodes because it doesn’t have Railcraft’s dependencies.

Reply:
I want all of the mixins to run in dev; and since we mixin railcraft (Currently, repo is likely moving it to RC proper), we need RC's deps to run in dev (buildcraft, etc), hence the transitive = true.  While I agree with explicit deps, it's a run time dep, not a compile time dep thus I"m not adding BC and such.